src/DeviceInput/InputDevices/webDeviceInputSystem.ts

```diff
@@ -636,8 +636,6 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
                         deviceEvent.previousState = 1;
                         this.onInputChanged(deviceEvent);
 
-                        this._unregisterDevice(DeviceType.Touch, deviceSlot);

```

Comment:
why was this removed?

Reply:
So I had originally removed this because I had a rolling system of active ids and wanted to prevent the any unnecessary array deletions for touch.  After doing a couple of edge tests, I've found that I need to have something still trigger the onDeviceDisconnected functionality so I'm going to add a direct call to it in place of this.

Reply:
Are we confident the array won't grow too big?

Reply:
The touch array should never grow any larger than the value of _maxTouchPoints which should be no greater than 10 (platform-dependent).